techniques/failures/F61.html

```diff
@@ -3,7 +3,7 @@
                     automatic update that the user cannot disable from within the content</h1><section class="meta"><p class="id">ID: F61</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
       <p>General </p>
    </section><section id="description"><h2>Description</h2>
-      <p> This document describes a failure that occurs when the content in the main <a href="/TR/WCAG/#dfn-viewport">viewport</a> viewport is automatically updated, and there is no option for a user to disable this behavior. </p>
+      <p> This document describes a failure that occurs when the content in the main <a>viewport</a> viewport is automatically updated, and there is no option for a user to disable this behavior. </p>
```

Comment:
I'm assuming this is only supposed to be the word "viewport" once?
```suggestion
      <p> This document describes a failure that occurs when the content in the main <a>viewport</a> is automatically updated, and there is no option for a user to disable this behavior. </p>
```